.gitignore

```
@@ -0,0 +1,2 @@
+/vendor/
+ERROR_LOG
```

Thanks. I wasn't sure whether or not it was a good idea to commit the file, but it makes sense to ignore it at least. We should probably also the DEBUG_LOG file to the ignore list. That one should never be committed, but it can be created when developers are trying to understand what happens in the code. ;-)
```suggestion
ERROR_LOG
DEBUG_LOG
```

---

I have now pulled this out into it's own commit 02b7a5e3e98d4d3eb84ad7ec2b65e73ef0ff01b2